Update `nodejs-express` example to use `@now/node` builder

TooTallNate opened the pull request:
Because express returns the HTTP handler as the `app` object,
we can use the `@now/node` builder and directly export the `app`
object. This is how it is done in the Express + Now 2.0 blog post.

timothyis commented:
@TooTallNate could you do me a favour and add this build to the primary `now.json` file at the root of the repo? It will act as a CI for this build.

rauchg commented:
This example would be much much better if it followed the monorepo structrure outlined by @TejasQ 

https://zeit.co/blog/serverless-express-js-lambdas-with-now-2

Instead of a single monolithic express

TooTallNate commented:
Ya good call @rauchg. I've updated it so that it produces two lambdas, one for each page.

@timothyis I added the build rules to the root `now.json` as well
